app/src/main/java/com/yeonae/chamelezone/view/home/adapter/HomePlaceRvAdapter.kt

```diff
@@ -54,29 +51,35 @@ class HomePlaceRvAdapter(private val currentLatitude: Double, private val curren
                     keyword.text = "${keyword.text}, $it"
                 }
             }
+
             if (place.savedImageName.isNotEmpty()) {
+                Logger.d("imageList ${place.savedImageName}, ${place.placeNumber}, ${place.memberNumber}")
                 val image = IMAGE_RESOURCE + place.savedImageName[0]
+                Logger.d("image $image")
                 placeImg.glideImageSet(image, placeImg.measuredWidth, placeImg.measuredHeight)
             }
 
+
             itemView.setOnClickListener {
                 itemClickListener.onItemClick(itemView, adapterPosition, place)
+
             }
 
             itemView.apply {
-                btn_like.isChecked = place.likeStatus
-                btn_like.setOnClickListener {
-                    likeButtonListener.onLikeClick(place, btn_like.isChecked)
+                if (place.memberNumber != 0) {
+                    btn_like.isChecked = place.likeStatus
+                    btn_like.setOnClickListener {
+                        likeButtonListener.onLikeClick(place, btn_like.isChecked)
+                    }
+                } else {
+                    btn_like.isChecked = place.likeStatus
                 }
```

Comment:
```suggestion
                btn_like.isChecked = place.likeStatus
                if (place.memberNumber != 0) {
                    btn_like.setOnClickListener {
                        likeButtonListener.onLikeClick(place, btn_like.isChecked)
                    }
                }
```
 btn_like.isChecked = place.likeStatus 라는 동작은 if 던 else 던 똑같이 수행하는데, 굳이 두번 작성할 필요가 없어보입니다.

Reply:
수정했습니다
10a60d5